src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml

```diff
@@ -74,21 +74,45 @@ tests:
                         EpochKey2: "hex:d2d1d2d3d4d5d6d7d8d9dadbdcdddedf",
                         EpochStartTime2: 2220002,
                     }
+    - label:
+          "Step 1b: TH sends KeySetWrite command in the GroupKeyManagement
+          cluster to DUT using a key that is pre-installed on the TH.
+          GroupKeySet fields are as follows: GroupKeySetID: 0x01a1
+          GroupKeySecurityPolicy: TrustFirst (0) EpochKey0:
+          a0d1d2d3d4d5d6d7d8d9dadbdcdddedf EpochStartTime0: 2220000 EpochKey1:
+          b1d1d2d3d4d5d6d7d8d9dadbdcdddedf EpochStartTime1: 2220001 EpochKey2:
+          c2d1d2d3d4d5d6d7d8d9dadbdcdddedf EpochStartTime2: 2220002"
+      cluster: "Group Key Management"
+      command: "KeySetWrite"
+      arguments:
+          values:
+              - name: GroupKeySet
+                value:
+                    {
+                        GroupKeySetID: 0x01a1,
+                        GroupKeySecurityPolicy: 0,
+                        EpochKey0: "hex:a0d1d2d3d4d5d6d7d8d9dadbdcdddedf",
```

Comment:
i thought we created some macros for those epoch keys.. this is a nit

Reply:
This one definitely wouldn't have one - I don't think it's used anywhere else.

Reply:
let me see if I can find the macro and add the other keys

Reply:
oh, wait, this is the yaml. Probably in the test plan. Does YAML do macros?

Reply:
I don't think we currently support global macros in the yaml, so I'm going to close this for now. I think this probably relates to the test plan